lib/cylc/scheduler.py

```diff
@@ -906,8 +906,8 @@ def run(self):
 
         proc_pool = SuiteProcPool.get_inst()
 
-        fs_check_period = datetime.timedelta(minutes=10)
-        next_fs_check = datetime.datetime.utcnow() + fs_check_period
+        fs_check_period = 600.0
```

Comment:
Given the nature of this variable, you may want to put this at the top of the class/module, and mark it as a class/module level constant. It is also worth adding a trailing `# seconds` comment to indicate that we are talking about 600.0 seconds.